src/main.cpp

```diff
@@ -281,9 +281,10 @@ int main(int argc, char * argv[])
         SetConsoleTitleA("Mixxx Debug Messages");
     }
 #endif
+    UINT oldCodePage;
```

Comment:
@default-kramer added 06cd51e39c51f6004071f45f8ca720fa701f5f1b in response to your comment. It would be worth initializing the uint but it isn't used without initialization now.